cjs/html/text-element.js

```diff
@@ -10,7 +10,11 @@ class TextElement extends HTMLElement {
 
   toString() {
     const outerHTML = toString.call(this.cloneNode());
-    return outerHTML.replace(/></, `>${this.textContent}<`);
+    // Use `join` in here to prevent unexpected special replacement patterns
+    // See also 
+    // https://github.com/WebReflection/linkedom/issues/292
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_string_as_the_replacement
+    return outerHTML.split('><').join(`>${this.textContent}<`);
```

Comment:
I would just add () => in front of the replacement, it feels faster and easier on both RAM and GC